### PR TITLE
Grafana provisioning content (datasources, dashboards)

### PR DESCRIPTION
**Pull request by PremyslCerny**
TODO:
- [x] Influx Auth token must be inserted.

Follow ups:

- [ ] Swap metrics from `swap` and not from `mem`
- [ ] Add "rate of swap" into system-level overview and detail
- [ ] Docker metrics are not working (?)

### CODE REVIEW

File: Site/ASAB Maestro/Files/grafana/provisioning/datasources.yaml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+     defaultBucket: BUCKET
+     organization: ORG
+  # <string> json object of data that will be encrypted.
+  secureJsonData:
+    token: TOKEN
+  version: 1
+  # <bool> allow users to edit datasources from the UI.
+  editable: true
+
+- name: InfluxDBTelegraf
```
Comment (ateska) [inline Site/ASAB Maestro/Files/grafana/provisioning/datasources.yaml:50]: We don't have two buckets in Influx.

**Review by ateska — CHANGES_REQUESTED** [review]

File: Site/ASAB Maestro/Files/grafana/provisioning/datasources.yaml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  - name: InfluxDB
+    orgId: 1
+  - name: InfluxDBTelegraf
+    orgId: 1
+
+# list of datasources to insert/update depending
+# whats available in the database
+datasources:
+  # <string, required> name of the datasource. Required
+- name: InfluxDB
```
Comment (ateska) [inline Site/ASAB Maestro/Files/grafana/provisioning/datasources.yaml:15]: Must be aligned with https://ndocs.teskalabs.com/logman.io/metrics/viewing-metrics/ 

... which _mainly_ means that all telemetry is in "InfluxDB" datasource.

Which is wrong too, b/c it names the technology.
But ... I give up that fight here.

**Review by ateska — CHANGES_REQUESTED** [review]

**Comment by ateska** [comment]
Here is a dump of my research:


https://grafana.com/docs/grafana/latest/developers/http_api/data_source/#create-a-data-source

```
{
  "uid": "lmio-influxdb-metrics",
  "orgId": 1,
  "name": "InfluxDB",
  "type": "influxdb",
  "typeLogoUrl": "public/app/plugins/datasource/influxdb/img/influxdb_logo.svg",
  "access": "proxy",
  "url": "http://lmio-XXX:8086",
  "user": "",
  "database": "",
  "basicAuth": false,
  "basicAuthUser": "",
  "withCredentials": false,
  "isDefault": true,
  "jsonData": {
    "dbName": "metrics",
    "httpHeaderName1": "Authorization",
    "httpMode": "POST"
  },
  "secureJsonFields": {
    "httpHeaderValue1": "Token YYY"
  },
  "version": 3,
  "readOnly": false
}
```

**Comment by PremyslCerny** [comment]
assigning to @eliska-n since she wanted to continue with it

**Comment by ateska** [comment]
Provisioning is working, TODOs has to be fixed.

**Review by eliska-n — APPROVED** [review]

**Pull request merged**